suite-native/module-trading/src/components/general/SearchableSheetHeader.tsx

```
@@ -0,0 +1,79 @@
+    onClose: () => void;
+    title: ReactNode;
+    onFilterFocusChange?: (isFilterActive: boolean) => void;
+    children?: ReactNode;
+    style?: NativeStyleObject;
+};
+
+export const FOCUS_ANIMATION_DURATION = 300 as const;
+
+const noOp = () => {};
+
+const wrapperStyle = prepareNativeStyle<{}>(({ spacings }) => ({
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)